client/src/components/Filter.jsx

```diff
@@ -100,11 +107,37 @@ const Filter = ({ filter, setFilter }) => {
                 {sector.toUpperCase()}
               </option>
             ))}
-          </select>
+          </FilterSelect>
         </>
       )}
-    </>
+    </FilterWrapper>
   );
 };
 
 export default Filter;
+
+const FilterWrapper = styled.div`
+  padding: 0.5rem;
+  position: relative;
+  height: 30%;
+  margin-bottom: 1rem;
+  display: flex;
+  gap: 5px;
+  flex-direction: column;
+  background-color: var(--color-light-gray);
+  border-radius: var(--border-radius);
+  box-shadow: var(--box-shadow);
+  &:hover {
```

Comment:
See my comment about the cursor above ;)

Reply:
Ok.